api_pagopa.yaml

```diff
@@ -0,0 +1,57 @@
+swagger: "2.0"
+info:
+  version: 1.0.0
+  title: Proxy API
+  description: Mobile and web proxy API gateway.
+host: localhost
+basePath: /api/v1
+schemes:
+  - https
+security:
+  - Token: []
+paths:
+  "/user":
+    x-swagger-router-controller: PagoPAController
+    get:
+      operationId: getUser
+      summary: Get user's data
+      description: Returns the user data needed by PagaPA proxy.
+      responses:
+        "200":
+          description: Found.
+          schema:
+            - $ref: "#/definitions/UserForPagoPA"
+          examples:
+            application/json:
+              email: "email@example.com"
+        "401":
+          description: Token null or invalid.
+        "500":
+          description: There was an error in retrieving the user data.
+          schema:
+            $ref: "#/definitions/ProblemJson"
+definitions:
+  EmailAddress:
+    $ref: "https://raw.githubusercontent.com/teamdigitale/digital-citizenship-functions/v0.40.0/api/definitions.yaml#/EmailAddress"
+  ProblemJson:
+    $ref: "https://raw.githubusercontent.com/teamdigitale/digital-citizenship-functions/v0.40.0/api/definitions.yaml#/ProblemJson"
+  UserForPagoPA:
+    title: UserForPagoPA
+    description: User data needed by PagaPA proxy.
```

Comment:
this is actually needed by the PagoPA Wallet backend, a component of PagoPA that handles users preferences (instead, PagoPA Proxy is a component of our infrastructure)